packages/block-library/src/comment-content/block.json

```diff
@@ -41,6 +41,9 @@
 				"padding": true
 			}
 		},
-		"html": false
+		"html": false,
+		"interactivity": {
+			"clientNavigation": true
+		}
```

Comment:
We talked about this one, but I don't remember our conclusions. Was this one supposed to be compatible? Do you remember?

Reply:
I think it was not compatible due to the CSS not being loaded when doing region navigation, right?

Reply:
Ok, let's keep it incompatible for now.